Makefile

```diff
@@ -203,6 +203,10 @@ format: install-goimports
 	@echo "> Formating code..."
 	$(if $(shell goimports -l -w -local github.com/buildpacks/lifecycle .), @echo Fixed formatting errors. Re-run && exit 1)
 
+tidy:
```

Comment:
I think it's worth adding this to `make unit/all/acceptance`.
Or maybe we can create another target that is running `lint format tidy` and use it when running `make unit/all/acceptance`.

Reply:
Added to test targets